pyproject.toml

```
@@ -5,179 +5,158 @@ build-backend = 'setuptools.build_meta'
+  'Operating System :: POSIX',
+  'Programming Language :: Python :: 3.10',
+  'Programming Language :: Python :: 3.11',
+  'Programming Language :: Python :: 3.12',
+  'Programming Language :: Python :: 3.9',
+  'Topic :: Scientific/Engineering :: Information Analysis',
 ]
 dependencies = [
-    'matplotlib>=3.0.1',
-    'numpy>=1.21.0',  # minimum typing support
-    'pillow',
-    'pooch',
-    'scooby>=0.5.1',
-    'vtk',  # keep without version constraints
-    'typing-extensions',
+  'matplotlib>=3.0.1',
+  'numpy>=1.21.0',     # minimum typing support
+  'pillow',
+  'pooch',
+  'scooby>=0.5.1',
+  'typing-extensions',
+  'vtk',               # keep without version constraints
 ]
 dynamic = ['version']
 
 [project.optional-dependencies]
 all = ['pyvista[colormaps,io,jupyter]']
-colormaps = [
-    'cmocean',
-    'colorcet',
-]
-io = [
-    'imageio',
-    'meshio>=5.2'
-]
+colormaps = ['cmocean', 'colorcet']
+io = ['imageio', 'meshio>=5.2']
 jupyter = [
-    'ipywidgets',
-    'jupyter-server-proxy',
-    'nest_asyncio',
-    'trame>=2.5.2',
-    'trame-client>=2.12.7',
-    'trame-server>=2.11.7',
-    'trame-vtk>=2.5.8',
-    'trame-vuetify>=2.3.1',
+  'ipywidgets',
+  'jupyter-server-proxy',
+  'nest_asyncio',
+  'trame-client>=2.12.7',
+  'trame-server>=2.11.7',
+  'trame-vtk>=2.5.8',
+  'trame-vuetify>=2.3.1',
+  'trame>=2.5.2',
 ]
-
-pinned = [  # Pinned versions of core dependencies
-    'matplotlib<3.9.3',
-    'numpy<2.2.0',
-    'pillow<11.1.0',
-    'pooch<1.9.0',
-    'scooby<0.11.0',
-    'vtk<9.4.0',
-    'typing-extensions<4.13.0',
-]
-typing= [
-    'pyvista[pinned]',
-    'numpy>=2.0.0',
-    'mypy<1.14.0',
-    'npt-promote==0.1',
+# Pinned versions of core dependencies
+pinned = [
+  'matplotlib<3.9.3',
+  'numpy<2.2.0',
+  'pillow<11.1.0',
+  'pooch<1.9.0',
+  'scooby<0.11.0',
+  'typing-extensions<4.13.0',
+  'vtk<9.4.0',
 ]
+typing = ['mypy<1.14.0', 'npt-promote==0.1', 'numpy>=2.0.0', 'pyvista[pinned]']
 test = [
-    'pyvista[pinned]',
-    'cmocean<4.0.4',
-    'colorcet<3.2.0',
-    # Embreex does not work with arm-based macs
-    "embreex<2.17.8; sys_platform != 'darwin' or platform_machine != 'arm64'",
-    'hypothesis<6.115.6',
-    'imageio<2.37.0',
-    'imageio-ffmpeg<0.6.0',
-    'ipython<9.0.0',
-    'ipywidgets<9.0.0',
-    'meshio<5.4.0',
-    'nest_asyncio<1.6.1',
-    'numpydoc==1.8.0',
-    'pyanalyze<=0.13.1',
-    'pytest<8.4.0',
-    'pytest-cov<6.1.0',
-    'pytest-memprof<0.3.0',
-    'pytest-pyvista==0.1.8',
-    'pytest-xdist<3.7.0',
-    'scipy<1.14.2',
-    'sphinx<8.2.0',
-    'sphinx-book-theme<1.2.0',
-    'sphinx-gallery<0.19.0',
-    'sphinx_design<0.7.0',
-    'sympy<1.14.0',
-    'tqdm<4.67.0',
-    'trame>=2.5.2,<3.7.1',
-    'trame-vtk>=2.5.8,<2.8.12',
-    'trame-vuetify>=2.3.1,<2.7.2',
-    'trimesh<4.6.0',
-    'typing-extensions<4.13.0',
+  # Embreex does not work with arm-based macs
+  "embreex<2.17.8; sys_platform != 'darwin' or platform_machine != 'arm64'",
```

Flip the quote style so that this is properly sorted
```suggestion
  'embreex<2.17.8; sys_platform != "darwin" or platform_machine != "arm64"',
```